test/dialogs/partition_size_test.rb

```diff
@@ -3,35 +3,76 @@
 require "cwm/rspec"
 require "y2partitioner/dialogs/partition_size"
 
-describe Y2Partitioner::Dialogs::PartitionSize do
-  let(:disk) { double("Disk", name: "mydisk") }
-  let(:ptemplate) { double("partition template") }
-  let(:slots) { [] }
-
-  subject { described_class.new(disk, ptemplate, slots) }
-  before do
-    allow(Y2Partitioner::Dialogs::PartitionSize::SizeWidget)
-      .to receive(:new).and_return(term(:Empty))
+describe "Partition Size widgets" do
+  let(:ptemplate) do
+    double("partition template",
+      region:      region,
+      custom_size: Y2Storage::DiskSize.MiB(1))
```

Comment:
why using double and not class directly? I see no side-effects which requires mocking class.